### DISCUSSION TITLE
ttcli generating for gradle ???

### DISCUSSION THREAD

**Discussion by hzendejas** [wimdeblauwe/wimdeblauwe.com#7] [Ideas]
Is ttcli available for gradle??? If not, then any guide how init a tailwindcss, thymeleaf and spring for a gradle project ???

**Comment by grey-shirt** [comment]
Hey @hzendejas 

[bootify.io](https://bootify.io/) is a pretty nice (free) site to generate a gralde + thymeleaf project.

@wimdeblauwe [wrote about](https://www.wimdeblauwe.com/blog/2024/02/12/rapid-application-prototyping-with-bootify/) it last month too.

**Comment by wimdeblauwe** [comment]
There is a feature request for it at https://github.com/wimdeblauwe/ttcli/issues/19, but as I don't use Gradle, I have no interest in adding this to the tool. If somebody would contribute a PR, I would consider adding it. The answer of @grey-shirt is certainly a good way if you really want to use Gradle.
